ec2/sc-ec2-linux-jumpcloud-v1.0.0.yaml

```diff
@@ -436,6 +441,35 @@ Resources:
                   - "/usr/bin/aws ec2 create-tags --region $AWS_REGION --resources $ROOT_DISK_ID "
                   - "--tags Key=Name,Value=$EC2_INSTANCE_ID-root Key=cloudformation:stack-name,Value=$STACK_NAME "
                   - "Key=Department,Value=$DEPARTMENT Key=Project,Value=$PROJECT Key=OwnerEmail,Value=$OWNER_EMAIL"
+        TagInstance:
+          files:
+            /opt/sage/bin/apply_name_tag.sh:
+              content: |
+                #! /bin/bash
+
+                set -xe
+
+                source /opt/sage/bin/instance_env_vars.sh
+                RESOURCE_ID=${STACK_ID##*/}
```

Comment:
why you need `##*/`?

Reply:
I am trimming everything in the ARN that is not the part after the last forward-slash. I'm using bash native string manipulation to do so. If I try use the whole ARN it breaks the SearchQuery.

Reply:
ahh ok.